### PR TITLE
[fix|sde-v2.0.10] hot fixes for bpn validation, documents update

### PR DESCRIPTION
**Pull request by adityagajbhiye9**
## Description
- BPN validation enhancements in consumer data page.
- Add BPN flow enhancement in provider, access policy section.
- Documentation updated.
- About page added.
- Script added to generate values in about page.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by almadigabor** [comment]
According to @adityagajbhiye9 it fixes the following issues:

- https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/issues/22
- https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/issues/23
- https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/issues/24
- https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/issues/28
- https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/issues/27

File: DEPENDENCIES [hunk 1]
```diff
@@ -169,7 +169,7 @@ npm/npmjs/-/cssom/0.3.8, MIT, approved, clearlydefined
 npm/npmjs/-/cssom/0.4.4, MIT, approved, clearlydefined
 npm/npmjs/-/cssstyle/2.3.0, MIT, approved, clearlydefined
 npm/npmjs/-/csstype/3.1.1, MIT, approved, clearlydefined
-npm/npmjs/-/cx-portal-shared-components/1.4.0, Apache-2.0 AND CC-BY-4.0 AND OFL-1.1, approved, #9102
+npm/npmjs/-/cx-portal-shared-components/1.5.0, , restricted, clearlydefined
```
Comment (almadigabor) [inline DEPENDENCIES:172]: You have now a restricted component, we need to open an IP issue ticket for that.

File: src/assets/about/content.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "managed-simple-data-exchanger-frontend",
+  "repositoryPath": "https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend",
+  "license": "Apache-2.0",
+  "licensePath": "https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/blob/2.0.9/LICENSE",
```
Comment (almadigabor) [inline src/assets/about/content.json:5]: You should do this in a dynamic way, you won't remember updating the tag for each release. The changelog already describes version 2.0.10 and the links here point to 2.0.9.

File: README.md [hunk 3]
```diff
@@ -1,58 +1,47 @@
...
+### Running the image from GitHub container registry
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+    export IMAGE=tractusx/sdefrontend:latest
+    docker pull $IMAGE
+    docker run --rm -d -p 3001:8080 --name sdefrontend $IMAGE
 
-## `npm eject`
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+## Notice for Docker image
```
Comment (almadigabor) [inline README.md:32]: You're still missing some info for the docker image notice section:

- Link to your repository on GitHub
- Direct link to the Dockerfile used to build your image
- Link to LICENCE file in your repo as 'Project License' (make clear, that this is the PROJECT licence, not an image license)

Take [this as an example](https://github.com/eclipse-tractusx/item-relationship-service/blob/main/README.md#notice-for-docker-image).

**Review by almadigabor — CHANGES_REQUESTED** [review]

File: src/assets/about/content.json (hunk 2 above)
Comment (adityagajbhiye9) [reply src/assets/about/content.json:5]: sure.

File: README.md (hunk 3 above)
Comment (adityagajbhiye9) [reply README.md:32]: added the notice for docker.

File: DEPENDENCIES (hunk 1 above)
Comment (almadigabor) [reply DEPENDENCIES:172]: @adityagajbhiye9  Created an IP issue ticket [here](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/10235). Please keep track of it's status and update your dependencies file when it's approved.

Comment (adityagajbhiye9) [reply DEPENDENCIES]: we updated dependency file.

File: src/assets/about/content.json (hunk 2 above)
Comment (adityagajbhiye9) [reply src/assets/about/content.json]: dynamic values generation scripts added.

**Comment by adityagajbhiye9** [comment]
@ajithsimons  Please review and confirm.

**Review by ajithsimons — APPROVED** [review]
Looks good to me

File: DEPENDENCIES [hunk 4]
```diff
@@ -169,7 +169,7 @@ npm/npmjs/-/cssom/0.3.8, MIT, approved, clearlydefined
 npm/npmjs/-/cssom/0.4.4, MIT, approved, clearlydefined
 npm/npmjs/-/cssstyle/2.3.0, MIT, approved, clearlydefined
 npm/npmjs/-/csstype/3.1.1, MIT, approved, clearlydefined
-npm/npmjs/-/cx-portal-shared-components/1.5.0, , restricted, clearlydefined
```
Comment (almadigabor) [inline DEPENDENCIES]: We should wait until the ticket is evaluated before merging this. We cannot create a release with a restricted dependency. If it is really important for some reason to merge this to main, we can do that without creating a release @adityagajbhiye9

File: DEPENDENCIES [hunk 5]
```diff
@@ -169,7 +169,7 @@ npm/npmjs/-/cssom/0.3.8, MIT, approved, clearlydefined
 npm/npmjs/-/cssom/0.4.4, MIT, approved, clearlydefined
 npm/npmjs/-/cssstyle/2.3.0, MIT, approved, clearlydefined
 npm/npmjs/-/csstype/3.1.1, MIT, approved, clearlydefined
-npm/npmjs/-/cx-portal-shared-components/1.4.0, Apache-2.0 AND CC-BY-4.0 AND OFL-1.1, approved, #9102
+npm/npmjs/-/cx-portal-shared-components/1.5.0, Apache-2.0 AND (Apache-2.0 AND LicenseRef-scancode-unknown) AND CC-BY-4.0 AND OFL-1.1, restricted, #10235
```
Comment (ajithsimons) [inline DEPENDENCIES:172]: @almadigabor  ticket is created in the eclipse side already. Won't this be enough to go ahead for releasing? We have done like this before.

**Comment by almadigabor** [comment]
Hey @adityagajbhiye9, please update the dependencies file, the IP ticket has been approved.

**Comment by adityagajbhiye9** [comment]
@almadigabor  dependency updated.

**Review by almadigabor — APPROVED** [review]

**Pull request merged**